nautobot/extras/tests/test_webhooks.py

```diff
@@ -155,7 +158,11 @@ class FakeResponse:
                 site.save()
 
                 serializer = SiteSerializer(site, context={"request": None})
-                snapshots = get_snapshots(site, ObjectChangeActionChoices.ACTION_CREATE)
+                oc = ObjectChange.objects.filter(
+                    changed_object_type=ContentType.objects.get_for_model(site),
+                    changed_object_id=site.pk,
+                ).first()
```

Comment:
This is repeated enough times that a helper function might be better?

Reply:
Do you think `utilities/testing/utils.py` or do you think this could be useful elsewhere, maybe for job hook receivers?

Reply:
If there's no utility for it outside of testing, a test util is sufficient.